app/i18n/nl/index.php

```diff
@@ -40,6 +40,7 @@
 		'mark_all_read' => 'Markeer alles als gelezen',
 		'mark_cat_read' => 'Markeer categorie als gelezen',
 		'mark_feed_read' => 'Markeer feed als gelezen',
+		'mark_selection_unread' => 'Mark selection as unread',	//TODO
```

Comment:
Markeer selectie als ongelezen

Reply:
Alright, I'll wait until several of these are merged (or at the very least until tomorrow) before I do that. :-)